test/jdk/java/awt/Window/TranslucentJAppletTest/TranslucentJAppletTest.java

```diff
@@ -23,11 +23,12 @@
 
 /*
  * @test %I% %E%
+ * @key headful
  * @bug 6683728
  * @summary Tests that a JApplet in a translucent JFrame works properly
  * @author Kenneth.Russell@sun.com: area=Graphics
```

Comment:
we need to remove the author tag. We only keep it if it's contributed from community. Also, better to remove the %I% %E% wildcard from @test tag which are not used nowadays..